src/Engine/Trigger/TriggerRepeat.cpp

```diff
@@ -67,11 +52,11 @@ void TriggerRepeat::OnLeave() {
     }
 }
 
-std::string TriggerRepeat::GetName() {
+const std::string& TriggerRepeat::GetName() const {
     return name;
 }
 
-void TriggerRepeat::SetName(std::string value) {
+void TriggerRepeat::SetName(std::string& value) {
```

Comment:
The parameter can be const because we are not modifying it (when we save it a copy is made, and we pass by reference to not make an unnecessary copy in the parameter).